Sources/ProjectSpec/SpecValidation.swift

```diff
@@ -53,9 +53,9 @@ extension Project {
             }
         }
 
-        for package in localPackages {
-            if !(basePath + Path(package).normalize()).exists {
-                errors.append(.invalidLocalPackage(package))
+        for (name, package) in localPackages {
+            if !(basePath + Path(package.path).normalize()).exists {
```

Comment:
Maybe you can use `relativePath` extension instead.

https://github.com/yonaskolb/XcodeGen/blob/e9239f83bd3725c462d42219ba45761a88ffe1e1/Sources/Core/PathExtensions.swift#L22

Reply:
I think it is intended to no use `relativePath` here. 
It is because this part is validating the specified file is existing or not by using an absolute path, so other code of this function is also using absolute path.